integration/update_compat_test.go

```diff
@@ -120,7 +120,7 @@ func testUpdateCompat(t *testing.T, testCases map[string]updateCompatTestCase) {
 								// Skip updates that could not be performed due to Tigris schema validation.
 								var e mongo.CommandError
 								if errors.As(targetErr, &e) && e.Name == "DocumentValidationFailure" {
-									if e.HasErrorCodeWithMessage(121, "json schema validation failed for field") {
+									if errorTextContains(e, "json schema validation failed for field", "does not validate with") {
```

Comment:
Is error code `121` no longer valid for validation?

Reply:
It does but the options for invalid texts now can be different... Let me fix it to check the error code and texts separately.

Reply:
fixed